launcher_scripts/conf/training/gpt3/8b.yaml

```diff
@@ -0,0 +1,211 @@
+run:
+  name: gpt3_8b
+  results_dir: ${base_results_dir}/${.name}
+  time_limit: "1-00:00:00"
+  dependency: "singleton"
+
+trainer:
+  num_nodes: 16
+  devices: 8
+  accelerator: gpu
+  precision: bf16
+  logger: False # logger provided by exp_manager
+  enable_checkpointing: False
+  replace_sampler_ddp: False
+  max_epochs: null
+  max_steps: 75000 # consumed_samples = global_step * global_batch_size
```

Comment:
Again need to adjust max steps based on 512 batch size and maybe 2048 seq length.